tests/pallas/pallas_test.py

```diff
@@ -1222,7 +1222,11 @@ def kernel(x_ref, y_ref, o_ref):
       o_ref[...] = f(x_ref[...], y_ref[...])
 
     x = jnp.array([1, 3, -4, -6, 2, 5, 4, -7]).astype(dtype)
-    y = jnp.array([3, 1, -4, -5, 2, -2, 2, 4]).astype(dtype)
+    if (f == jnp.bitwise_left_shift):
```

Comment:
Nit: please remove the parens and use `is` instead of `==`.

Reply:
done